### PR TITLE
Update conf enac imav

### PR DESCRIPTION
**Pull request by gautierhattenberger**
update enac conf and imav2022 flight plans

collection of small corrections:
- default mode is NAV for rover steering
- no rover throttle when killed
- xbee link doesn't send data to modem before end of config
- fix alt in server (geoid/ellipsoid)
- support GPS_INT in sd2log
- don't increment position in follow module
- use correct angles for hybrid

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c [hunk 1]
```diff
@@ -144,17 +147,6 @@ static void send_fp(struct transport_tx *trans, struct link_device *dev)
                               &autopilot.flight_time);
 }
 
-static void send_body_rates_accel(struct transport_tx *trans, struct link_device *dev)
```
Comment (fvantienen) [inline sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:147]: This was accidentally removed I guess while rebasing

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c [hunk 2]
```diff
@@ -210,7 +202,6 @@ void autopilot_firmware_init(void)
   register_periodic_telemetry(DefaultPeriodic, PPRZ_MSG_ID_ROTORCRAFT_FP, send_fp);
   register_periodic_telemetry(DefaultPeriodic, PPRZ_MSG_ID_ROTORCRAFT_FP_MIN, send_fp_min);
   register_periodic_telemetry(DefaultPeriodic, PPRZ_MSG_ID_ROTORCRAFT_CMD, send_rotorcraft_cmd);
-  register_periodic_telemetry(DefaultPeriodic, PPRZ_MSG_ID_BODY_RATES_ACCEL, send_body_rates_accel);
```
Comment (fvantienen) [inline sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:213]: See above

**Review by fvantienen — CHANGES_REQUESTED** [review]
Nice I love the fixes!

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c (hunk 1 above)
Comment (gautierhattenberger) [reply sw/airborne/firmwares/rotorcraft/autopilot_firmware.c]: sure, my fault

**Comment by gautierhattenberger** [comment]
@fvantienen I think I fixed the issues here

File: sw/ground_segment/tmtc/link.ml [hunk 3]
```diff
@@ -527,12 +533,10 @@ let () =
     begin
       ignore (Glib.Timeout.add !status_msg_period (fun () -> send_status_msg (); true));
       ignore (Glib.Timeout.add (!status_msg_period / 3) (fun () -> update_ms_since_last_msg (); true));
-      if !uplink then begin
```
Comment (fvantienen) [inline sw/ground_segment/tmtc/link.ml:530]: This is also a fix which was reverted

**Review by fvantienen — CHANGES_REQUESTED** [review]
Just a tiny small mistake still, next to that ok for merging!

File: sw/ground_segment/tmtc/link.ml (hunk 3 above)
Comment (gautierhattenberger) [reply sw/ground_segment/tmtc/link.ml]: dammit :( should be fixed now

**Review by fvantienen — APPROVED** [review]

**Pull request merged**